src/features/import/l10n/messageIds.ts

```diff
@@ -242,7 +242,7 @@ export default makeMessages('feat.import', {
           'This problem exists on rows {commaRows} and {lastRow}.'
         ),
         manyRows: m<{ additionalRows: number; commaRows: string }>(
-          'This problem exists on rows {commaRows} and {additionalRows} additional rows.'
+          'This problem exists on rows {commaRows} and {additionalRows, plural, =1 {1 additional row} other {# additional rows}}.'
```

Comment:
woop woop!

Reply:
Yes, it makes sense so that it shows which row has error. I will create an issue for it tomorrow!